pom.xml

```diff
@@ -88,6 +88,7 @@
         <allure-maven.version>2.10.0</allure-maven.version>
         <skipFunctionalTests>false</skipFunctionalTests>
         <skipUnitTests>false</skipUnitTests>
+        <iabgpp-encoder.version>3.0.10</iabgpp-encoder.version>
```

Comment:
This shouldn't be a part of tests. Haven't we already added that in a production branch?

Reply:
Not yet